app/views/layouts/listnav/_host.html.haml

```diff
@@ -77,6 +77,12 @@
               ems_infra_path(@record.ext_management_system),
               :title => _("Show this parent %{provider} for this %{host}") % {:host => host_title, :provider => ui_lookup(:table => "ems_infra")})
 
+        - if role_allows?(:feature => "physical_server_show") && !@record.physical_server.nil?
+          %li
+            = link_to("#{ui_lookup({:table => "physical_servers"})}: #{@record.physical_server.name}",
```

Comment:
In addition to this, you will also need to consider a case when a host does not have a Physical Server associated with it and then display `Physical Server: None` in the listnav without a link.

```ruby
- if role_allows?(:feature => "physical_server_show") && @record. physical_server.nil?
  = li_link(:if => false, :text => "#{ui_lookup({:table => "physical_servers"})}: #{_('None')}")
```